solutions/logging/client-experimentation/README.md

```diff
@@ -38,46 +37,55 @@ This package has no sub-packages.
 
 ## Usage
 
-1.  Clone the package:
+1. Clone the package:
+
     ```shell
     kpt pkg get https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit.git/solutions/logging/client-experimentation@${VERSION}
     ```
+
     Replace `${VERSION}` with the desired repo branch or tag
     (for example, `main`).
 
-1.  Move into the local package:
+1. Move into the local package:
+
     ```shell
     cd ".//solutions/logging/client-experimentation/"
     ```
 
-1.  Edit the function config file(s):
+1. Edit the function config file(s):
```

Comment:
I would like to see these have their own heading 2-3 so we can href them from other parts of the docs - but this can be done as part of an edit that is required via a targeted jira later

Reply:
the content between the markers `<!-- BEGINNING OF PRE-COMMIT-BLUEPRINT DOCS HOOK:BODY -->` is auto generated by the kpt command generate docs, we should avoid customizing these

Reply:
@danielsvensson-ssc now that you have modified the rules, I was expecting that your PR wouldn't include any changes to generated files